source/sass/blocks/header.scss

```
@@ -1,17 +1,63 @@
 .header {
-  margin: 0;
-  padding: 11px 0 17px 0;
+  @media (min-width: $width-desktop) {
+    &--home {
+      $hero-desktop-width: 1044px;
+      background: url("../../img/hero-desktop.jpg") no-repeat calc(50% + $hero-desktop-width / 2),
```

```suggestion
      background: url("../img/hero-desktop.jpg") no-repeat calc(50% + $hero-desktop-width / 2),
```
Так как набор символов `../../` говорит браузеру о переходе на 2 уровня вверх, а это уже будет за пределами корневой директории сайта. Локально, в bowsersynk работает, так как этот плагин обрабатывает подобные ситуации, но в реальном сайта скорее не будет работать, чем работать. Можете проверить на примере Github Pages (нужно нажать кнопку публикации проекта на странице с заданием).